### PR TITLE
added grid resources as well as one homework per day

### PR DESCRIPTION
**Pull request by raphamontenegro**
We need to decide which option is better for today's homework

### CODE REVIEW

**Comment by raphamontenegro** [comment]
homework order is wrong. Not sure how to fix it (26 and 27 are misplaced)

File: _homework/web-development/portfolio-creation.md [hunk 1]
```diff
@@ -8,9 +8,19 @@ due: Oct 2, 2018
 Tasks
 --------
 
-- Fork Rapha's [repository](https://github.com/raphamontenegro)
+OPTION 1
```
Comment (nevanscott) [inline _homework/web-development/portfolio-creation.md]: Let's remove this one and just use Option 2

File: _homework/web-development/portfolio-final.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: Finish your webpage
+assigned: Oct 4, 2018
+due: Oct 4, 2018
```
Comment (nevanscott) [inline _homework/web-development/portfolio-final.md]: Change to Oct 5

**Review by nevanscott — CHANGES_REQUESTED** [review]
Just a couple quick changes

**Review by nevanscott — APPROVED** [review]

**Pull request merged**